internal/kernel/t_aio/store.go

```diff
@@ -178,6 +205,13 @@ type Result struct {
 	TimeoutPromises            *AlterPromisesResult
 	TimeoutDeleteSubscriptions *AlterSubscriptionsResult
 	TimeoutCreateNotifications *AlterNotificationsResult
+
+	// LOCKS
```

Comment:
All good to keep this comment, but if we do let's add for all other types to be consistent